examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,113 @@
+## Multi-Tenant logging from EKS using Fluent-bit, Amazon MSK and  Amazon OpenSearch
+
+**Let us first start by explaining some of the key terms we will refer throught the solution .**
```

Comment:
This is long enough that personally I think would be most readable if it was split up into a series of sections with proper headers- and this could be the first header, just `### Terminology`